XmlRpcValue.h header with was moved in kinetic. Use a ros-comm header to
trigger its inclusion indirectly.

--- include/jrl/dynamics/urdf/parser.hh~	2014-04-29 12:06:55.000000000 +0200
+++ include/jrl/dynamics/urdf/parser.hh	2017-04-04 17:30:01.610317660 +0200
@@ -27,7 +27,7 @@
 # include <abstract-robot-dynamics/humanoid-dynamic-robot.hh>
 # include <jrl/dynamics/dynamicsfactory.hh>
 
-# include "XmlRpcValue.h"
+# include "ros/param.h" /* for XmlRpcValue.h header with was moved in kinetic */
 
 namespace jrl
 {
